config/purposes/heron_lthr_384.yml

```diff
@@ -47,6 +47,7 @@ LTHR-384 Lib PCR 1:
   :tag_layout_templates:
     - TSsc384-PCR1-nCoV-2019/V3/B
```

Comment:
Nothing to do with this PR but I am wondering why we left the v3 option here when we moved to v4, and whether we should remove the v3 and v4 when we go to v5. It doesn't make much sense to me to allow them to use old tag plates under a new v5 submission. In fact it sounds like a very bad idea. Can ask Scott G.